tests/foreman/ui/test_containerimagetag.py

```diff
@@ -27,12 +27,16 @@
 
 @pytest.fixture(scope="module")
 def module_org():
-    return entities.Organization().create()
+    org = entities.Organization().create()
+    yield org
+    org.delete()
 
 
 @pytest.fixture(scope="module")
 def module_product(module_org):
-    return entities.Product(organization=module_org).create()
+    product = entities.Product(organization=module_org).create()
+    yield product
+    product.delete()
```

Comment:
Deleting anything just waits time given we do not keep the Satellite after testing.